jetbrains-core/tst/software/aws/toolkits/jetbrains/utils/RunConfigTestUtils.kt

```diff
@@ -59,7 +59,13 @@ fun executeRunConfiguration(runConfiguration: RunConfiguration, executorId: Stri
             val executionEnvironment = executionEnvironmentBuilder.build()
 
             val listener = object : OutputListener() {
+                override fun processWillTerminate(event: ProcessEvent, willBeDestroyed: Boolean) {
+                    println("Received processWillTerminate for ${event.processHandler}")
```

Comment:
Do we need to log both processWillTerminate and processTerminated?

Reply:
trying to understand the lifecycle here and if it's possible for one to be called without the other